bigtable-client-core-parent/bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/adapters/read/TestRowAdapter.java

```diff
@@ -66,59 +64,49 @@ public void adaptResponse_oneRow() {
-    byte[] value5 = "value5".getBytes();
- 
+
     ByteString key = ByteString.copyFromUtf8("key");
     final long ts1Micros = 54321L;
     final long ts2Micros = 12345L;
     final long ts1Millis = ts1Micros / 1000;
     final long ts2Millis = ts2Micros / 1000;
-    Row row = Row.newBuilder()
-        .setKey(key)
-        .addFamilies(Family.newBuilder()
-            .setName(family1)
-            .addColumns(Column.newBuilder()
-                .setQualifier(ByteString.copyFrom(qualifier1))
-                .addCells(Cell.newBuilder() // First cell.
-                    .setTimestampMicros(ts1Micros)
-                    .setValue(ByteString.copyFrom(value1)))
-                .addCells(Cell.newBuilder() // Same family, same column, but different timestamps.
-                  .setTimestampMicros(ts2Micros)
-                  .setValue(ByteString.copyFrom(value2)))
-                .addCells(Cell.newBuilder() // With label
-                    .setValue(ByteString.copyFromUtf8("withLabel"))
-                    .addLabels("label")))
-            .addColumns(Column.newBuilder()
-                .setQualifier(ByteString.copyFrom(qualifier2))
-                .addCells(Cell.newBuilder() // Same family, same timestamp, but different column.
-                    .setTimestampMicros(ts1Micros)
-                    .setValue(ByteString.copyFrom(value3)))))
-        .addFamilies(Family.newBuilder()
-            .setName(family2)
-            .addColumns(Column.newBuilder()
-                .setQualifier(ByteString.copyFrom(qualifier1))
-                .addCells(Cell.newBuilder() // Same column, same timestamp, but different family.
-                    .setTimestampMicros(ts1Micros)
-                    .setValue(ByteString.copyFrom(value4))))
-            .addColumns(Column.newBuilder()
-                .setQualifier(ByteString.copyFrom(qualifier2))
-                .addCells(Cell.newBuilder() // Same timestamp, but different family and column.
-                    .setTimestampMicros(ts1Micros)
-                    .setValue(ByteString.copyFrom(value5)))))
-        .build();
+    String label = "label";
+    List<String> labelList = Collections.emptyList();
 
-    Result result = instance.adaptResponse(row);
-    assertEquals(5, result.rawCells().length);
+    ImmutableList.Builder<RowCell> rowCells = ImmutableList.builder();
+    rowCells.add(RowCell.create(family1, ByteString.copyFrom(qualifier1), ts1Micros, labelList,
+        ByteString.copyFrom(value1)));
+    rowCells.add(RowCell.create(family1, ByteString.copyFrom(qualifier2), ts2Micros, labelList,
+        ByteString.copyFrom(value2)));
+    rowCells.add(RowCell.create(family2, ByteString.copyFrom(qualifier2), ts2Micros, labelList,
+        ByteString.copyFrom(value3)));
+    rowCells.add(RowCell.create(family2, ByteString.copyFrom(qualifier1), ts1Micros, labelList,
+        ByteString.copyFrom(value4)));
+
+    //Added duplicate row, should be ignored
```

Comment:
Added duplicate row; should be ignored. (use the same format for line 90. 'Should' is capitalized in one and not in the other. )